lib/board.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+class Board
+  # Is this saving
+
+  attr_accessor :cells
+
+  def initialize(cells = Array.new(9," "))
+    @cells = cells
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.

Reply:
You'll soon learn about a programming concept called DRY (Don't repeat yourself).  In this case, recreating an empty board in 2 methods right next to each other might not seem like a big deal. However, with larger applications and when you are working in a team, you don't want to be using the same or similar code to accomplish the same thing in multiple places.  If all those methods need the same code for the app to function correctly and you or someone changes it in just in the places they remember that app will break.  It is best practice to define some functionality by itself then call it where you need it in your app.  If the app needs to change you only need to change the code in one place and everything will continue to function.

Reply:
Alright so it's like saying "Don't put all your eggs in one basket", and one method does only one thing. I think I kind of understand better but I will learn more about it later.